src/app/pages/Info/Info.tsx

```diff
@@ -7,8 +7,9 @@ import style from './Info.module.css';
 
 export default function Info(): JSX.Element {
   const { id }: { id: string } = useParams();
+  const idNbr = parseInt(id, 10);
```

Comment:
whut's dis? Abbreviations are difficult to read

Reply:
please check line 10 at Info.tsx. UseParams returns strings and I need a number for the filter in line 12. Thank you